fabtests/pytest/efa/test_rma_pingpong.py

```diff
@@ -0,0 +1,34 @@
+from efa.efa_common import efa_run_client_server_test
+import pytest
+
+
+@pytest.mark.parametrize("operation_type", ["writedata", "write"])
+@pytest.mark.parametrize("iteration_type",
+                         [pytest.param("short", marks=pytest.mark.short),
+                          pytest.param("standard", marks=pytest.mark.standard)])
+def test_rma_pingpong(cmdline_args, iteration_type, operation_type, completion_semantic, memory_type):
+    command = "fi_rma_pingpong -e rdm"
```

Comment:
`memory_type` includes hmem memory type, please add something like

```
if memory_type != "host_to_host":
    pytest.skip("no hmem memory support")
```

Reply:
Update: I think only ft_rma_poll_buf doesn't support hmem memory, which is for write op. We may refine this condition based on that.